docs/changelog.md

```diff
@@ -6,6 +6,7 @@
 
 - Support TSM-MobileNetV2
 - Support TANet
+- Refactor config: change the position of `train_cfg` and `test_cfg`
```

Comment:
Refactorings sometimes cause trouble for old users, so let's don't put it in highlight